web/src/pages/index.tsx

```diff
@@ -16,20 +16,26 @@ const actionBoxMockup: CallToActionBoxProps[] = [
       'Bjerk driver med utvikling av desktop- og mobilapplikasjoner. Vi leverer digitale løsninger basert på moderne teknologi og metoder.',
     linkText: 'Les mer',
     linkTo: '/services',
+    image: 'Code.svg',
```

Comment:
Maybe we can reference a component here instead? Look at my other comment for more on that.

```suggestion
    image: <CodeIllustration />,
```